test/transaction_repository_test.rb

```diff
@@ -1,127 +1,127 @@
   end
 
   def test_all
-
-    assert_equal 3, transaction_repository.all.size
+    assert_equal 3, transaction_repo.all.size
   end
 
   def test_random
-
-    assert_class = Transaction, transaction_repository.random
+    assert_equal Transaction, transaction_repo.random.class
   end
 
   def test_find_by_transaction_id
+    transaction = transaction_repo.find_by_id(1)
 
-    transaction = transaction_repository.find_by_id(1)
     assert_equal 1, transaction.id
   end
 
   def test_find_by_invoice_id
+    transaction = transaction_repo.find_by_invoice_id(2)
 
-    transaction = transaction_repository.find_by_invoice_id(2)
     assert_equal 2, transaction.invoice_id
   end
 
   def test_find_by_credit_card_number
+    transaction = transaction_repo
+      .find_by_credit_card_number("4354495077693036")
 
-    transaction = transaction_repository.find_by_credit_card_number('4354495077693036')
-    assert_equal '4354495077693036', transaction.credit_card_number
+    assert_equal "4354495077693036", transaction.credit_card_number
   end
 
   def test_find_by_credit_card_expiration
+    transaction = transaction_repo
+      .find_by_credit_card_expiration("2012-03-27 14:54:09")
 
-    transaction = transaction_repository.find_by_credit_card_expiration('2012-03-27 14:54:09')
-    assert_equal '2012-03-27 14:54:09', transaction.credit_card_expiration
+    assert_equal "2012-03-27 14:54:09", transaction.credit_card_expiration
   end
 
   def test_find_by_result
+    transaction = transaction_repo.find_by_result("success")
 
-    transaction = transaction_repository.find_by_result('success')
-    assert_equal 'success', transaction.result
+    assert_equal "success", transaction.result
   end
 
   def test_find_all_by_transaction_id
+    transactions = transaction_repo.find_all_by_transaction_id(2)
+    transactions1= transaction_repo.find_all_by_transaction_id(3)
 
-    transactions = transaction_repository.find_all_by_transaction_id(2)
-    transactions1= transaction_repository.find_all_by_transaction_id(3)
     assert_equal 1, transactions.size
     assert_equal 1, transactions1.size
   end
 
   def test_find_all_by_invoice_id
+    transaction = transaction_repo.find_all_by_invoice_id(3)
 
-    transaction = transaction_repository.find_all_by_invoice_id(3)
     assert_equal 1, transaction.size
-
   end
 
   def test_find_all_by_credit_card_number
+    transactions = transaction_repo.find_all_by_credit_card_number("")
+    transactions1 = transaction_repo
+      .find_all_by_credit_card_number("4654405418249632")
 
-    transactions = transaction_repository.find_all_by_credit_card_number('')
-    transactions1 = transaction_repository.find_all_by_credit_card_number('4654405418249632')
     assert_equal 0, transactions.size
     assert_equal 1, transactions1.size
   end
 
   def test_find_all_by_credit_card_expiration
+    transactions = transaction_repo
+      .find_all_by_credit_card_expiration("2012-03-27 14:54:09")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.